### ISSUE TITLE
A Little Bit Of Balance? 

### ISSUE THREAD

**Issue by Crazy-Elipsis** [Kitisplode/minecraftMod-golemFirstStone#13]
Helloo! After testing and messing around with this nice mod for a bit, I soon realized that if I ever host a private server with this mod players could end up having a whole army of golems in no time! Specially for how cheap they are to make!

An army of golems can kill almost anything, including strong modded entities too. Here's an example image of how a bunch of plank golems are totally destroying the warden.
They're dealing so much damage and so much knockback that the warden can't attack, not even shoot his laser.

![image](https://github.com/Kitisplode/minecraftMod-golemFirstStone/assets/148471848/1e7d1bc4-ed83-44ef-96f8-b0de8eaee66d)

I'm currently working on a private modpack (Not sure if I'll end up releasing it to the public or not), but I find myself a bit stuck on how to actually balance the use of these new golems because the player can have an unlimited amount of them and also make them for cheap, totally breaking the game balance upon bossfights and whatnot.

I tried looking inside the mod if there was any file that would let me change a few values to replace the making of the golems, my idea was to replace the carved pumpkin for something that is a bit harder to get, like a more rare/expensive block like using the mod's golem's heads which I can nerf its crafting recipe to make it slighthly more accessible. Something like this:

![image](https://github.com/Kitisplode/minecraftMod-golemFirstStone/assets/148471848/1daa5946-8309-4377-ba3e-9c3206e09924)

Another weird idea I came up with would be making the shears inaccessible upon completing [X] requirements??? Not really sure. I'm still puzzled about it so that's why I came here for help! 

**TLDR:**

- The player has the ability to control as many golems as they want, could be troublesome on server hosting or game balance
- Small golems are really cheap to make and they're a bit too powerful in big quantities
- (Not related and also not sure if intentional) Big golem structures are located about 10k - 20k or 30k blocks away from the spawn for some reason

Looking forward for a response, anything helps! Thank you and keep it up with this wonderful mod! <3
